content/en/docs/setup/getting-started/index.md

```diff
@@ -110,7 +110,7 @@ Follow these steps to get started with Istio:
 
 ## Deploy the sample application {#bookinfo}
 
-1.  Deploy the [`Bookinfo` sample application](/docs/examples/bookinfo/):
+1.  Follow steps in the page [`Bookinfo` sample application](/docs/examples/bookinfo/) to deploy sample application:
```

Comment:
I don't think we want to change the getting started page to tell users to go to another document to do things. The getting started doc is supposed to be a simple, self contained, set of instructions to make it easy for users to start looking at Istio.

I see why the current request-routing instructions are a little confusing (hidden). My suggestion would be to remove the warning on line 46 and put the warning under each tab (`Istio APIs` and `Gateway API`) instead.

Then you can change the wording under the `Istio APIs` tab to make it clear that it's asking you to create the destination rules:
```
If you haven’t already applied destination rules, follow the instructions in [define the service versions](/docs/examples/bookinfo/#define-the-service-versions).
```